src/hooks/use-create-editor.ts

```diff
@@ -78,8 +80,16 @@ export const useCreateEditor = () => {
     return () => stepsExchanger.stop()
   }, [dispatch, stepsExchanger])
 
-  const getCapabilities = memoize((project, user, permittedActions) =>
```

Comment:
I wonder why we used memoize here before.

Reply:
I'm not sure, to be honest.